pkg/constants/constants.go

```diff
@@ -48,6 +48,10 @@ const (
 	// The event uses the error(s) as the reason.
 	JobCreationFailedReason = "JobCreationFailed"
 
+	// Event reason used when a Headless Service creation fails.
+	// The event uses the error(s) as the reason.
```

Comment:
```suggestion
	// The event uses the error(s) as the message.
```

Can you make the same change for `JobCreationFailedReason` above as well? This is a typo.